mlflow/models/model.py

```diff
@@ -175,6 +180,7 @@ def get_model_info(self):
             saved_input_example_info=self.saved_input_example_info,
             signature_dict=self.signature.to_dict() if self.signature else None,
             utc_time_created=self.utc_time_created,
+            mlflow_version=self.mlflow_version,
```

Comment:
to make this backward compatible, perhaps add in a conditional check?

```suggestion
            mlflow_version=self.mlflow_version if self.mlflow_version else None,
```

Reply:
This is code that runs within the instance that will always have the mlflow_version property, even if it's loaded from an older file in which case it will be just None so no real need to add that conditional check. Happy to add it though if you think it's really necessary